src/coreclr/inc/pgo_formatprocessing.h

```diff
@@ -361,6 +361,44 @@ bool ReadInstrumentationSchemaWithLayout(const uint8_t *pByte, size_t cbDataMax,
     });
 }
 
+
+// Return true if schemaTable entries are a subset of the schema described by pByte, with matching entries in the same order.
+// Also updates offset of the matching entries in schemaTable to those of the pByte schema.
+//
+inline bool ComparePgoSchemaCompatible(const uint8_t *pByte, size_t cbDataMax, ICorJitInfo::PgoInstrumentationSchema* schemaTable, size_t cSchemas)
```

Comment:
I was a little surprised that this mutates the schema. Maybe rename it to something like `SetDataForSchemaSubsequence` or similar?

Reply:
How about `CheckIfPgoSchemaIsCompatibleAndSetOffsets`?